company.el

```diff
@@ -641,7 +641,9 @@ commands in the `company-' namespace, abort completion."
 (defcustom company-show-numbers nil
   "If enabled, show quick-access numbers for the first ten candidates."
   :type '(choice (const :tag "off" nil)
-                 (const :tag "on" t)))
+                 (const :tag "left" 'left)
+                 (const :tag "right" 'right)
```

Comment:
Why `right`? Just use `t` for this.

Reply:
It's for some user who imagines symmetric configurations... I'll remove it